### DISCUSSION TITLE
Create Column not included in the query

### DISCUSSION THREAD

**Discussion by michele-grifa** [rappasoft/laravel-livewire-tables#898] [Q&A] [answered]
Hi, is possible to create a boolean column that is not included in the query?
I have some custom logic on the row properties and i want to display true/false based on that.

**Comment by gRoberts84** [comment]
I'm interested in the answer to this. 

I would like to create custom columns that don't rely on an actual value in the model or database to work. 

The only way I've been able to make it work is to include a column that isn't actually needed and then format it how I want, which isn't ideal.

**Comment by lrljoe** [comment, marked as answer]
Setup the Column as a Label one, and it won't try to add it to the query.

```php
Column::make('My one off column')
->label(
   fn ($row, Column $column) => 'Whatever Function You Would Like'
),
```

**Reply by michele-grifa** [reply]
It worked.

**Reply by lrljoe** [reply]
Great, if you can mark this as answered then it'll help me keep track of who still needs help, and help others find a solution to their issue in future!
